draft-ietf-tls-tls13.md

```diff
@@ -3690,23 +3690,22 @@ In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the following cipher suites:
 
 ~~~~
-    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
-    TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    TLS_AES_128_GCM_SHA256
 ~~~~
 
-These cipher suites MUST support both digital signatures and key exchange
-with secp256r1 (NIST P-256) and SHOULD support key exchange with X25519
-{{RFC7748}}.
-
 A TLS-compliant application SHOULD implement the following cipher suites:
 
 ~~~~
-    TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
-    TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256
-    TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
-    TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+    TLS_AES_256_GCM_SHA384
+    TLS_CHACHA20_POLY1305_SHA256
 ~~~~
```

Comment:
Now that things are simpler, this is overly complex formatting. I suggest simply having things inline (as was done in previous TLS docs). e.g.:

"
In the absence of an application profile standard specifying otherwise, a
TLS-compliant application MUST implement TLS_AES_128_GCM_SHA256
cipher suite. A TLS-compliant application SHOULD also implement the
TLS_AES_256_GCM_SHA384 and TLS_CHACHA20_POLY1305_SHA256
cipher suites.
"

If this change isn't made, there's a nit that the first list is now just one entry, so "suites" should be "suite".